File: Modul4/Soal 5/PRAK405-2310817210020-MuhammadHasbiyanRusyadi.c
#include<stdio.h>
int main() {
    int a, z;
    scanf("%d %d", &a, &z);
    int total = 0;
    for (int i = 1; i <= a; i++) {
        int tot = 0;
        for (int j = i; j >= 1; j--) {
            printf("(%d * %d)", j, z);
            tot += j * z;
            if (j > 1) {
                printf(" + ");}}
        total += tot;
        printf(" = %d\n", tot);}
printf("%d", total);}

File: Modul4/Soal 5/PRAK405-2310817210020-MuhammadHasbiyanRusyadi.py
a, z =map(int, input().split())
total = 0
for i in range(1, a + 1):
    tot = 0
    for j in range(i, 0, -1):
        print(f"({j} * {z})", end="")
        tot += j * z
        if j > 1:
            print(" + ", end="")
    total += tot
    print(f" = {tot}")
print(total)
